test/optimus/4kpbmc/ValidateOptimus.wdl

```diff
@@ -0,0 +1,272 @@
+workflow ValidateOptimus {
```

Comment:
When/if we migrate to a new repo, we need to figure out a way to do this without code duplication.